app/services/admin/reopen_induction_period.rb

```diff
@@ -31,13 +31,18 @@ def reopen_induction_period!
 
     def check_can_reopen_period!
       check_induction_period_is_complete!
+      check_induction_period_has_outcome!
```

Comment:
This wasn't technically in-scope, but it felt odd to restrict things in the UI without making the same changes here.